src/file_organizer/services/copilot/rules/executor.py

```
@@ -0,0 +1,419 @@
+                    action_type=action_type.value,
+                    status="skipped",
+                    message="tag action has no storage backend yet",
+                )
+            return ExecutionResult(
+                file_path=str(source),
+                rule_name=rule.name,
+                action_type=action_type.value,
+                status="failed",
+                message="unsupported action",
+            )
+        except Exception as exc:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)